### DISCUSSION TITLE
how to make beamer theme works in VScode-Marp?

### DISCUSSION THREAD

**Discussion by kongdd** [marp-team/marp#359] [Q&A]
Any idea about how to make beamer theme works in VScode-Marp?
> https://github.com/n-ari/marp-theme-nordic-beamer
> See slides/demo/index.md, or [built demo slide on GitHub pages](https://n-ari.github.io/marp-theme-nordic-beamer/demo/).

Thanks and regards

**Comment by cunhapaulo** [comment]
Try seaching for cunhapaulo/marpstyle at Github.com.

It may interest you.

https://github.com/cunhapaulo/marpstyle


Em qua, 19 de out de 2022 12:24, Dongdong Kong ***@***.***>
escreveu:

**Comment by yhatt** [comment]
It seems to provide as [the customized Marp engine](https://github.com/n-ari/marp-theme-nordic-beamer/blob/master/marp.config.js), _not theme CSS_. Marp for VS Code only supports custom theme CSS so you must use [Marp CLI](https://github.com/marp-team/marp-cli) instead, [as described by the theme author.](https://github.com/n-ari/marp-theme-nordic-beamer#usage)

Or try to contact the author in order to be made the theme provide as CSS file.
